prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,116 @@ public async Task<ActionResult<Organization>> CreateOrganization(CreateOrganizat
             );
         }
 
+        // POST: api/Organizations/claim
+        /// <summary>
+        /// Claim an existing Organization.
+        /// </summary>
+        [HttpPost("claim", Name = nameof(ClaimOrganization))]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<int>), StatusCodes.Status200OK)]
+
+        public async Task<ActionResult<Organization>> ClaimOrganization(ClaimOrganizationViewModel claimOrganization)
+        {
+            if (!await _partyService.PartyExistsAsync(claimOrganization.PartyId, PartyType.SigningAuthority))
+            {
+                return BadRequest("Could not claim an organization, the passed in SigningAuthority does not exist.");
+            }
+
+            var organization = await _organizationService.ClaimOrganizationAsync(claimOrganization.PartyId, claimOrganization.PEC, claimOrganization.ClaimDetail);
+
+            return Ok(organization);
+        }
+
+        // GET: api/Organizations/claim
+        /// <summary>
+        /// Gets organization claim count by a given search criteria.
+        /// </summary>
+        [HttpGet("claim", Name = nameof(GetOrganizationClaimCount))]
```

Comment:
Bit of a nuance here with nomenclature: the word "claim" can be either a verb or a noun. In REST, resources are generally nouns (that you do HTTP verbs to), they are collections, and are plural in the URL. Actions are verbs, are generally singular, and really only make sense with POST (how do you verb a verb?).

If there was only a POST endpoint that was performing the "claim" action, I could see these endpoints using the singular. However, if you are doing a GET against it, claim is now a _noun_, and you are getting a collection of them. It makes more sense to me to see these endpoints with "claim" being a noun; where we are creating, approving, and getting lists of them. To that end, I think these URLs should be /claims rather than /claim.